java/hibernate/src/main/java/com/yugabyte/hibernatedemo/server/BasicHttpServer.java

```diff
@@ -68,20 +71,33 @@ private static void handleProductRequest(final HttpExchange exchange) throws IOE
 
         if (exchange.getRequestMethod().equals("POST")) {
         	handleCreateProduct(exchange);
-        } else {
+        } else if(exchange.getRequestMethod().equals("DELETE")){
+            handleDeleteProduct(exchange);
+        }else {
```

Comment:
nit: missing space before `else`. Please check formatting, in general, in the modified/newly added code.